### PR TITLE
Plocate support as an alternative

### PR DESCRIPTION
**Pull request by clexanis**
## Pull Request type

Please check the type of change your PR introduces:

- [ ] Bugfix
- [x] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build-related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

Currently, Debian *plocate* is set as `Replace` and `Breaks` *mlocate*.
But since *openmediavault-locate* specify *mlocate* as a `Depends`:

- If *plocate* is already installed, installing the plugin will remove *plocate*.
- If the plugin is installed (with *mlocate*), installing will remove *mlocate* **and** the plugin.

## What is the new behavior?

- The plugin itself is installed with *mlocate* by default (Web UI tested, obviously).
- *plocate* can be installed beforehand, it won't remote it and will use *plocate* in Web UI (tested).
- The plugin can be installed alongside *plocate* (`apt install openmediavault-locate plocate`), it won't trow an error.
- Switching between *mlocate* and *plocate* is possible, but since the config file is the same, *apt* will ask about merging.

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

In conclusion, can we say It just works? ( ͡° ͜ʖ ͡°)

### CODE REVIEW

**Comment by ryecoaaron** [comment]
If plocate is going to replace mlocate, it seems better to put plocate first in the dependency line so that apt will install plocate when neither is installed.

**Comment by clexanis** [comment]
The goal wasn't to use `plocate` by default, but to be able to others users who would want to.

Since I likely don't have your level of competence and experience, I don't think I should decide what should be used as default (and I'm not the maintainer). 

But, If you think it would be better, I can set it as default.

**Comment by ryecoaaron** [comment]
plocate is faster but mlocate is more secure (only shows files the user has access to). Since the plugin is running as root, seems like plocate should be the first choice for tye extra speed. I will give plocate a try to see if the speed is noticeable.

**Comment by clexanis** [comment]
I will try to make some measurements  too, but keep in mind that it would be made on a server with ~~too much~~ a lot of files, so it could need some hours (days?) just for building indexes.

**Comment by clexanis** [comment]
Update: As some of my drive have just starting returning errors at the same time (`187 Reported_Uncorrect`), I wouldn't be able to make any tests soon (or it would be biased).

**Comment by ryecoaaron** [comment]
We will just leave the mlocate default.  It can always be changed at a later date.
